airplanedb.py

```diff
@@ -178,3 +177,101 @@ def reset_db(self):
+        db = MySQLdb.connect(host=self.host, 
+                             user=self.user, 
+                             passwd=self.pw, 
+                             db=self.db)
+
+        update_customer_query = """UPDATE CUSTOMER
+                                   SET %s = '%s'
+                                   WHERE C_ID = %s""" % (field, new_value, custID)
+                                   
+        cursor = db.cursor()
+        try:
+            cursor.execute(update_customer_query)
+            db.commit()
+            print("Update Customer Success")
+        except:
+            print("Update Customer Failed")
+            db.rollback()
+        
+        db.close()
+
+#==============================================================================
+#   function: add_baggage
+#   description: adds an instance of baggage to BAGGAGE table
+#==============================================================================
+    def add_baggage(self, bag_ID, cust_ID, bag_weight):
+        db = MySQLdb.connect(host=self.host, 
+                             user=self.user, 
+                             passwd=self.pw, 
+                             db=self.db)
+
+        add_baggage_query = """INSERT INTO BAGGAGE(B_ID,C_ID,B_WEIGHT)
+                                VALUES('%s','%s','%s')""" % (bag_ID, cust_ID, bag_weight)
+
+        cursor = db.cursor()
+        try:
+            cursor.execute(add_baggage_query)
+            db.commit()
+            print("Add Baggage Success")
+        except:
+            print("Add Baggage Failed")
+            db.rollback()
+        
+        db.close()
+
+#==============================================================================
+#   function: add_customer
+#   description: adds an instance of customer to CUSTOMER table
+#==============================================================================
+    def add_customer(self, cust_name, cust_age, cust_email, cust_phone):
+         db = MySQLdb.connect(host=self.host, 
+                             user=self.user, 
+                             passwd=self.pw, 
+                             db=self.db)
+
+         add_customer_query = """INSERT INTO CUSTOMER(C_NAME, C_AGE, C_EMAIL, C_PHONE)
+                                 VALUES('%s','%s','%s','%s')""" % (cust_name, cust_age,
+                                 cust_email, cust_phone)
+
+         cursor = db.cursor()
+         try:
+             cursor.execute(add_customer_query)
+             db.commit()
+             print("Add Customer Success")
+         except:
+             print("Add Customer Failed")
+             db.rollback()
+        
+         db.close()
+
+#==============================================================================
+#   function: add_frequent_flier
+#   description: adds a new frequent flier instance to FREQUENTFLIER table
+#==============================================================================    
+    def add_frequent_flier(self, cust_ID, miles):
```

Comment:
we should have an update_frequent_flier function for every time the customer has logged more miles in their FF account?